extras/rya.geoindexing/geo.common/src/main/java/org/apache/rya/indexing/accumulo/geo/OptionalConfigUtils.java

```diff
@@ -81,7 +82,17 @@ public static boolean getUseGeoTemporal(final Configuration conf) {
      * {@code null} if it doesn't exist.
      */
     public static GeoIndexerType getGeoIndexerType(final Configuration conf) {
-        return conf.getEnum(GEO_INDEXER_TYPE, null);
+    	String confType[] = conf.getStrings(GEO_INDEXER_TYPE);
```

Comment:
there should be a get() that has a default value so you don't need to do a null check on confType